pkgs/development/tools/gomplate/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "gomplate";
+  version = "3.6.0";
+
+  # Git revision as defined here:
+  # https://github.com/hairyhenderson/gomplate/blob/eb2dbf65167e8bef7b62615c30431d75adc4e4b6/Makefile#L13
+  rev = "564f5588";
+
+  src = fetchFromGitHub {
+    owner = "hairyhenderson";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0b936fa92v4p50xmbi53y11a7s02bhzn6n7svy5vbbpryrfnjmf9";
+  };
+
+  preBuild = ''
+    export buildFlagsArray+=(
+      "-ldflags=
+        -w -s
+        -X ${goPackagePath}/v3/version.Version=${version}
+        -X ${goPackagePath}/v3/version.GitCommit=${rev}")
+  '';
+
+  goPackagePath = "github.com/hairyhenderson/gomplate";
```

Comment:
```suggestion
```

This is relevant only for `buildGoPackage`.

Reply:
True, but I'm using it on the `buildFlagsArray`. I could change the variable name if that makes the intention clearer.

Reply:
Then I guess it's not that important, keep it that way.